CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift

```diff
@@ -49,7 +49,53 @@ class GalleriesResponseHandlerSpec: QuickSpec {
                     }
                 }
             }
-
+            
+            it("Should return galleries owned by a user if .owned filter passed") {
+                guard let userIdentifier = TestConfiguration.testUserIdentifier
+                else { return }
+                
+                let request = GalleriesRequest(page: nil, perPage: nil, sort: nil, filter: .owned, userId: userIdentifier, query: nil)
+                let sender = TestComponentsFactory.requestSender
+                waitUntil(timeout: TestConfiguration.timeoutMedium) {
+                    done in
+                    _ = sender.login(TestConfiguration.username, password: TestConfiguration.password) {
+                        (error: Error?) -> Void in
+                        expect(error).to(beNil())
+                        _ = sender.send(request, withResponseHandler:GalleriesResponseHandler {
+                            (galleries: Array<Gallery>?, pageInfo: PagingInfo?, error: Error?) -> Void in
+                            expect(galleries).notTo(beNil())
+                            expect(error).to(beNil())
+                            expect(pageInfo).notTo(beNil())
```

Comment:
We're not testing if returned galleries indeed belong to the user. Maybe something like

`expect(galleries.first.owner.identifier).equals(userId)`

Reply:
Good point, will add it